src/__test__/PatchResolver.spec.js

```diff
@@ -164,6 +164,8 @@ describe('PathResolver', function () {
                     boundary,
                 });
 
+                resolver.handleChunk(`\r\n--${boundary}\r\n`);
```

Comment:
Can you also add a test where the preamble boundary is split into multiple chunks?

Reply:
Do any of these test a split somewhere in the middle of the boundary?